pkg/thumbnail/thumbnail.go

```diff
@@ -16,20 +16,27 @@ import (
 	"time"
 
 	"github.com/Chatterino/api/pkg/cache"
+	"github.com/Chatterino/api/pkg/config"
 	"github.com/Chatterino/api/pkg/resolver"
 	"github.com/Chatterino/api/pkg/utils"
 	"github.com/nfnt/resize"
 )
 
 var (
 	supportedThumbnails = []string{"image/jpeg", "image/png", "image/gif", "image/webp"}
+
+	cfg config.APIConfig
 )
 
 const (
 	// max width or height the thumbnail will be resized to
 	maxThumbnailSize = 300
 )
 
+func InitializeConfig(passedCfg config.APIConfig) {
```

Comment:
This is fine for now, but this to me says that the thumbnail resolver should really be its own struct that is initialized with this data

Reply:
Acknowledged. I'll make sure to note this and try transforming it to a struct that's initialized here in the future PR ![FarmingCommits](https://cdn.betterttv.net/emote/5ed57f8bfdee545e3065efcb/1x)